arduino-ide-extension/src/browser/boards/boards-config.tsx

```diff
@@ -306,7 +306,10 @@ export class BoardsConfig extends React.Component<
             type="search"
             value={query}
             className="theia-input"
-            placeholder="SEARCH BOARD"
+            placeholder={nls.localize(
+              'arduino/board/searchBoard',
+              'SEARCH BOARD'
```

Comment:
Can we use `search board` or `Search board` and transform it to upper case with CSS? See any example [here](https://jsfiddle.net/kittaakos/2ybfvuL9/).

I suggest we not force contributors helping with the translation to use the upper case. What do you think?

Reply:
I checked the other placeholder strings and found they are all sentence case, so I simply removed the cringe caps lock from the string entirely:

https://github.com/arduino/arduino-ide/pull/1341/commits/d250a1725e2eb9a23966cc32515dd3f09e7b4920

Reply:
I am OK with it. The upper casing comes from Create:

<img width="735" alt="Screen Shot 2022-08-22 at 09 59 37" src="https://user-images.githubusercontent.com/1405703/185870143-5d5f6068-11c1-40b2-bd81-792e21efe831.png">